src/tracer.ts

```diff
@@ -52,6 +55,60 @@ export class Tracer {
     // OpenTracing API methods
     // ---------------------------------------------------------------------- //
 
+    /**
+     * Gets the currently active span.
+     *
+     * @return the active Span. This is a shorthand for tracer.spanManager().active().
+     */
+    activeSpan(): Span | null {
```

Comment:
Similar to my comment below, I don't see the point of this since the alternative is not any more involved.